services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/DigitalGreenCertificateToCborMapping.java

```diff
@@ -44,42 +43,32 @@ public DigitalGreenCertificateToCborMapping(DigitalCovidCertificateClient digita
    *
    * @param ruleType - rule type for which the business rules will be retrieved.
    * @return - business rules
-   * @throws DigitalCovidCertificateException - exception thrown if anything happens while executing the logic.
-   *                                          Examples: business rules could not be fetched, one specific rule could not
-   *                                          be fetched, json validation fails for a specific rule etc. This exception
-   *                                          will propagate and will stop any archive to be published down in the
-   *                                          execution.
+   * @throws DigitalCovidCertificateException - thrown if json validation schema is not found or the validationfails for
```

Comment:
space missing (validationfails)

Reply:
I don't really figure out what to change here. The validation on the PR seems to pass.

Reply:
just a typo :)  thrown if json validation schema is not found or the validationfails for
```suggestion
   * @throws DigitalCovidCertificateException - thrown if json validation schema is not found or the validation fails for
```